packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.scss

```diff
@@ -55,7 +55,7 @@
   font-size: inherit;
   color: $ms-color-black;
   background-color: transparent;
-  flex: 1 1 0;
+  flex: 1 1 0px;
```

Comment:
Yes, its an Edge but the browser reads `1 1 0` as `1 1 0%` which breaks the layout, for other browsers its read correctly as `1 1 0px` so making it explicit.